x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/feature_access/essentials.cy.ts

```diff
@@ -55,9 +55,10 @@ describe(
       }
 
       // No access to response actions (except `unisolate`)
+      // TODO: update tests when `scan` is included in PLIs
       for (const actionName of RESPONSE_ACTION_API_COMMANDS_NAMES.filter(
-        (apiName) => apiName !== 'unisolate'
-      )) {
+        (apiName) => apiName !== 'scan'
+      ).filter((apiName) => apiName !== 'unisolate')) {
```

Comment:
I don't understand the double `.filter()` here, what do you think about the following? : 
```
for (const actionName of RESPONSE_ACTION_API_COMMANDS_NAMES.filter(
        (apiName) => apiName !== 'unisolate' && apiName !== 'scan'
 )) {

Reply:
Yeah that works the same way. I added another filter function just so that it is easier to delete later and not confuse with the logic for excluding `release` action from tests